cmd/replicate-update.go

```diff
@@ -171,7 +172,7 @@ func mainReplicateEdit(cliCtx *cli.Context) error {
 				dmReplicate = enableStatus
 			case "delete":
 				vDeleteReplicate = enableStatus
-			case "metadata-sync", "replica-metadata-sync":
+			case "replica-metadata-sync":
```

Comment:
@adfost , `replica-metadata-sync` was shortened to `metadata-sync` - we can just use metadata-sync here and in the help